### PR TITLE
Fix/ux papercuts transaction details issue #12295

### PR DESCRIPTION
**Pull request by igorms-cons**
Fixes:

Updating account details modal view

Issue: https://github.com/MetaMask/metamask-extension/issues/12295

### CODE REVIEW

**Comment by igorms-cons** [comment]
Also on the issue #12295 it was mentioned to pay attention to the format dat (24:40 instead of 00:40). Of course i did not tried to send a transaction at midnight but when i tested with a milli timestamp between 0:00am and 1:am, the format was good.

**Review by david0xd — DISMISSED** [review]
Looks good.

**Comment by mhasanagic** [comment]
Retested - verified by QA

**Review by david0xd — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Review by danjm — CHANGES_REQUESTED** [review]

File: ui/components/ui/sender-to-recipient/sender-to-recipient.component.js [hunk 1]
```diff
@@ -144,7 +142,6 @@ function RecipientWithAddress({
         onHidden={() => setAddressCopied(false)}
       >
         <div className="sender-to-recipient__name">
-          <span>{addressOnly ? `${t('to')}: ` : ''}</span>
```
[the remark this answers is not in the document]
Comment (igorms-cons) [reply ui/components/ui/sender-to-recipient/sender-to-recipient.component.js:147]: yes!

[the remark this answers is not in the document]
Comment (igorms-cons) [reply ui/components/ui/sender-to-recipient/sender-to-recipient.component.js:147]: On the figma the label "_to_" disappeared. 
You want me to keep it then?

[the remark this answers is not in the document]
Comment (igorms-cons) [reply ui/components/ui/sender-to-recipient/sender-to-recipient.component.js:147]: like I removed "_from_" line 74

[the remark this answers is not in the document]
Comment (danjm) [reply ui/components/ui/sender-to-recipient/sender-to-recipient.component.js:147]: oh, I thought that change wasn't being made in others parts of the app, but I can see that it is, okay, what you have is fine

**Review by danjm — APPROVED** [review]

**Pull request merged**